docker/images/outrigger/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:14.04
```

Comment:
That wouldn't allow a Mac user to build the node bits and push them to any of the docker linux images. Also thought that sticking to official builds on npm would be "better" in the long run.